app/src/main/java/de/westnordost/streetcomplete/quests/surface/Surface.kt

```diff
@@ -23,3 +23,24 @@ enum class Surface(val item:Item<String>) {
 }
 
 fun List<Surface>.toItems() = this.map { it.item }
+
+fun pavedSurfaces(): List<Surface> {
+    return listOf(
+            Surface.ASPHALT, Surface.CONCRETE, Surface.PAVING_STONES,
+            Surface.SETT, Surface.UNHEWN_COBBLESTONE, Surface.GRASS_PAVER,
+            Surface.WOOD, Surface.METAL
+    )
+}
```

Comment:
```suggestion
val PAVED_SURFACES = listOf(
            ASPHALT, CONCRETE, PAVING_STONES,
            SETT, UNHEWN_COBBLESTONE, GRASS_PAVER,
            WOOD, METAL
    )
```

Better as constant. Also, you can import `Surface.*`

Reply:
Somehow I never tested importing file into itself. It works beautifully.